### DISCUSSION TITLE
Cluster run issue (Windows, MS-MPI)

### DISCUSSION THREAD

**Discussion by tiborrrcfd** [nextfoam/baram#110] [General]
I get to a point where I get communication between machines, a new user is created on the cluster machine, but get this error:

> ERROR: Failed to get user credentials. Shutting down. Use -pwd <password> and -savecreds options to provide and save credentials.

How do you add or set these credentials? This topic is not really covered in the tutorial.

**Comment by tiborrrcfd** [comment]
Nevermind, here is the solution:

Configure MPI Service to Run Under Specific User Account:

- If you are using the Microsoft MPI (MS-MPI) implementation, you can configure the MPI service to run under a specific user account that has the necessary permissions.
- Open the Services management console (services.msc).
- Find the MSMPI Launch Service (the name might vary depending on your MPI version).
- Right-click and go to Properties.
- In the Log On tab, select This account and enter the desired user credentials.
- Restart the service to apply the changes.

![image](https://github.com/user-attachments/assets/4820c054-0f25-4241-b63a-b54cfb111f95)

**Comment by tiborrrcfd** [comment]
Now that I have solved the communication issue I am dealing with another issue - calculation 'starts', but nothing happens, no log is being written and no job starts on any node. Can anybody help with a basic tutorial to get BARAMCFD to run in cluster?

**Comment by kdg5424** [comment]
Hi @tiborrrcfd. This is Donggyu from NEXTFOAM.
Could you check your project folder again? There would be **log.txt** and **stderr.txt**.
These are the log file. So please check it and attach these files. After I check it, let me tell you how you should do it.

And there is no basic tutorial to run BARAM in cluster. It's one of our premium service. So we support it for the premium clients who paid.
It's FYI.

Thanks.

**Comment by kdg5424** [comment]
Hello @tiborrrcfd.

Have you tested the other MPI program on your cluster?
I recommend you to test the other MPI program like paraview or even simple, easy code like "Hello World!"
I just want to know your MPI setting is right.

Best regards

Donggyu kim
